src/DynamoCore/Graph/NodeGraph.cs

```diff
@@ -131,6 +133,13 @@ public static NoteModel LoadNoteFromXml(XmlNode note)
             return instance;
         }
 
+        public static ConnectorPinModel LoadPinFromXml(XmlNode pin)
```

Comment:
Also needs a summary. Could it be internal instead of public?

Reply:
Done.